### DISCUSSION TITLE
Reccomended diameter for preform?

### DISCUSSION THREAD

**Discussion by jamesy0ung** [universam1/iSpindel#622] [Q&A]
Hello, my local plastics suppliers only have 24 and 28mm inner diameter preforms, would these work for the iSpindel, I checked the [Parts list](https://github.com/universam1/iSpindel/blob/master/docs/Parts_en.md) and there are no dimensions mentioned for the preform.

**Comment by kmate** [comment]
I'm having an older v2.1 PCB which fits an ~33mm inner diameter tube.
